soco/music_services/music_service.py

```diff
@@ -0,0 +1,780 @@
+# -*- coding: utf-8 -*-
+
+"""Sonos Music Services interface.
+
+This module provides the MusicService class and related functionality.
+
+"""
+# pylint: disable=fixme
+
+from __future__ import unicode_literals
+
+import logging
+log = logging.getLogger(__name__)  # pylint: disable=C0103
+
+from xmltodict import parse
+import requests
+
+from soco import SoCo
+from soco.xml import XML
```

Comment:
After reading up on absolute vs. relative a bit, it seems that it is still not a clear recommendation for one or the other yet. So I guess maybe never mind about this and the other comment until we figure out what we prefer.